sgx-types/src/attr.rs

```diff
@@ -37,6 +37,12 @@ pub struct Attributes {
     xfrm: Xfrm,
 }
 
+impl Attributes {
+    pub const fn new(flags: Flags, xfrm: Xfrm) -> Self {
```

Comment:
Just curious why this is `const`.

Reply:
`const` functions can be evaluated at compile time. For example: https://doc.rust-lang.org/unstable-book/language-features/const-fn.html